### PR TITLE
chore(dependency): Update dependencies to what Fedora 23 has.

### PR DESCRIPTION
**Pull request by definite**
Highlight:

The xom:xom is renamed as com.io7m.xom

Following must change as well:
https://github.com/zanata/zanata-common/pull/20
https://github.com/zanata/zanata-server/pull/1110

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -1392,10 +1376,11 @@
       </dependency>
 
       <dependency>
-        <groupId>xom</groupId>
+        <!-- New group Id -->
+        <groupId>com.io7m.xom</groupId>
         <artifactId>xom</artifactId>
```
Comment (seanf) [inline pom.xml]: We should probably add xom:xom as a banned dependency, to make sure we always use the right one.

File: pom.xml [hunk 2]
```diff
@@ -179,13 +179,17 @@
...
     <findbugs.maven.version>3.0.1</findbugs.maven.version>
     <groovy.version>1.8.9</groovy.version>
     <guava.version>13.0.1</guava.version>
     <jackson.version>1.9.13</jackson.version>
-    <powermock.version>1.5.4</powermock.version>
+    <jaxb.api.version>2.2.12</jaxb.api.version>
+    <javadoc.version>2.9.1</javadoc.version>
+    <jxr.version>2.5</jxr.version>
+    <powermock.version>1.6.2</powermock.version>
+    <resteasy.version>3.0.13.Final</resteasy.version>
```
Comment (seanf) [inline pom.xml]: I don't think resteasy is used in the client any more, so we should remove resteasy from zanata-parent and put it all in the server pom.

**Comment by definite** [comment]
zanata-common-api still need it.
See https://github.com/zanata/zanata-api/blob/master/zanata-common-api/pom.xml

**Comment by seanf** [comment]
We could move it into both server and zanata-common-api to help with the
parent/server split, but never mind.

On 11 March 2016 at 15:29, Ding-Yi Chen <notifications@github.com> wrote:

> zanata-common-api still need it.
> See
> https://github.com/zanata/zanata-api/blob/master/zanata-common-api/pom.xml
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/zanata/zanata-parent/pull/52#issuecomment-195200693>.
>



-- 
Sean Flanigan

Principal Software Engineer
Globalisation Tools Engineering
Red Hat

**Comment by seanf** [comment]
:+1: except we should ban the xom:xom dependency

**Comment by seanf** [comment]
:+1: I suggest a squash-merge.

**Pull request merged**